src/presentation/service/account.go

```diff
@@ -0,0 +1,157 @@
+package service
+
+import (
+	"github.com/speedianet/os/src/domain/dto"
+	"github.com/speedianet/os/src/domain/useCase"
+	"github.com/speedianet/os/src/domain/valueObject"
+	accountInfra "github.com/speedianet/os/src/infra/account"
+	filesInfra "github.com/speedianet/os/src/infra/files"
+	serviceHelper "github.com/speedianet/os/src/presentation/service/helper"
+)
+
+type AccountService struct {
+}
+
+func NewAccountService() AccountService {
+	return AccountService{}
+}
+
+func (service AccountService) Read() ServiceOutput {
+	accountsQueryRepo := accountInfra.AccQueryRepo{}
+	accountsList, err := useCase.GetAccounts(accountsQueryRepo)
+	if err != nil {
+		return NewServiceOutput(InfraError, err.Error())
+	}
+
+	return NewServiceOutput(Success, accountsList)
+}
+
+func (service AccountService) Create(input map[string]interface{}) ServiceOutput {
+	requiredParams := []string{"username", "password"}
+	err := serviceHelper.RequiredParamsInspector(input, requiredParams)
+	if err != nil {
+		return NewServiceOutput(UserError, err.Error())
+	}
+
+	username, err := valueObject.NewUsername(input["username"])
+	if err != nil {
+		return NewServiceOutput(UserError, err.Error())
+	}
+
+	password, err := valueObject.NewPassword(input["password"])
+	if err != nil {
+		return NewServiceOutput(UserError, err.Error())
+	}
+
+	dto := dto.NewCreateAccount(username, password)
+
+	accQueryRepo := accountInfra.AccQueryRepo{}
+	accCmdRepo := accountInfra.AccCmdRepo{}
+	filesQueryRepo := filesInfra.FilesQueryRepo{}
+	filesCmdRepo := filesInfra.FilesCmdRepo{}
+
+	err = useCase.CreateAccount(
+		accQueryRepo,
+		accCmdRepo,
+		filesQueryRepo,
+		filesCmdRepo,
+		dto,
+	)
```

Comment:
We are not breaking lines like this anymore.

Reply:
Done.